src/frontend/src/pages/WorkPackageForm/WorkPackageForm.tsx

```diff
@@ -0,0 +1,76 @@
+import { UseMutationResult } from 'react-query';
+import { WbsNumber, WorkPackage, isGuest, isProject, wbsPipe } from 'shared';
+import WorkPackageFormView, { WorkPackageFormViewPayload } from './WorkPackageFormView';
+import { bulletsToObject } from '../../utils/form';
+import { useAllWorkPackages } from '../../hooks/work-packages.hooks';
+import LoadingIndicator from '../../components/LoadingIndicator';
+import ErrorPage from '../ErrorPage';
+import { useAllUsers } from '../../hooks/users.hooks';
+import { useSingleProject } from '../../hooks/projects.hooks';
+
+interface WorkPackageFormProps {
+  wbsNum: WbsNumber;
+  operation: (wbsNum: WbsNumber) => UseMutationResult;
+  exitActiveMode: () => void;
+  crId?: string;
+}
+
+const WorkPackageForm: React.FC<WorkPackageFormProps> = ({ wbsNum, operation, exitActiveMode, crId }) => {
+  const { data: users, isLoading: usersIsLoading, isError: usersIsError, error: usersError } = useAllUsers();
+  const {
+    data: project,
+    isLoading: projectIsLoading,
+    isError: projectIsError,
+    error: projectError
+  } = useSingleProject({ ...wbsNum, workPackageNumber: 0 });
+  const { data: workPackages, isLoading: wpIsLoading, isError: wpIsError, error: wpError } = useAllWorkPackages();
+  const { mutateAsync } = operation(wbsNum);
```

Comment:
this is a bit pedantic, but I would call this separately in the Create and Edit Form Files, and then just pass in the mutateAsync as a prop instead, this `= operation(wbsNum)` reads a bit confusing in isolation